packages/textarea/src/FzTextarea.vue

```diff
@@ -51,6 +54,8 @@ const props = withDefaults(defineProps<FzTextareaProps>(), {
   resize: "all",
 });
 
+const emit = defineEmits(["focus", "paste", "blur"]);
```

Comment:
can you use ts so that you can type the params?

Reply:
done in [36b9c02](https://github.com/fiscozen/design_system/pull/184/commits/36b9c0277107da59cd82d18c803bc3e478fdc0bd) 
I did not use the compact syntax because TS threw an error, we should investigate why it does.